python/ray/tune/utils/mock.py

```diff
@@ -1,4 +1,7 @@
+import logging
 import os
+
+import click
```

Comment:
can we lazily import click (inside on_step_begin)?

Reply:
Good point- done!